diff --git a/node_modules/gustwind/esm/gustwind-utilities/plugins.js b/node_modules/gustwind/esm/gustwind-utilities/plugins.js
index 5951034..3555c26 100644
--- a/node_modules/gustwind/esm/gustwind-utilities/plugins.js
+++ b/node_modules/gustwind/esm/gustwind-utilities/plugins.js
@@ -1,5 +1,5 @@
-import { join as joinPath } from "path";
-import process from "process";
+import { join as joinPath } from "node:path";
+import process from "node:process";
 const DEBUG = process.env.DEBUG === "1";
 async function importPlugins({ cwd, initialImportedPlugins, pluginDefinitions, outputDirectory, initLoadApi, mode, }) {
     const loadedPluginDefinitions = [];
