diff --git a/node_modules/gustwind/esm/gustwind-utilities/plugins.js b/node_modules/gustwind/esm/gustwind-utilities/plugins.js
index e2cfaeb..cb2b986 100644
--- a/node_modules/gustwind/esm/gustwind-utilities/plugins.js
+++ b/node_modules/gustwind/esm/gustwind-utilities/plugins.js
@@ -1,5 +1,5 @@
-import { join as joinPath } from "path";
-import process from "process";
+import { join as joinPath } from "node:path";
+import process from "node:process";
 const DEBUG = process.env.DEBUG === "1";
 async function importPlugins({ cwd, initialImportedPlugins, pluginDefinitions, outputDirectory, initLoadApi, mode, }) {
     const loadedPluginDefinitions = [];
diff --git a/node_modules/gustwind/esm/plugins/htmlisp-edge-renderer/mod.js b/node_modules/gustwind/esm/plugins/htmlisp-edge-renderer/mod.js
index 04f7d27..55b01d7 100644
--- a/node_modules/gustwind/esm/plugins/htmlisp-edge-renderer/mod.js
+++ b/node_modules/gustwind/esm/plugins/htmlisp-edge-renderer/mod.js
@@ -1,4 +1,4 @@
-import process from "process";
+import process from "node:process";
 import breezewind from "../../breezewind/mod.js";
 import { applyUtilities } from "../../breezewind/applyUtility.js";
 import * as breezeExtensions from "../../breezewind/extensions.js";
diff --git a/node_modules/gustwind/esm/plugins/meta/mod.js b/node_modules/gustwind/esm/plugins/meta/mod.js
index 5291fcc..315a11d 100644
--- a/node_modules/gustwind/esm/plugins/meta/mod.js
+++ b/node_modules/gustwind/esm/plugins/meta/mod.js
@@ -1,4 +1,4 @@
-import * as path from "path";
+import * as path from "node:path";
 const plugin = {
     meta: {
         name: "gustwind-meta-plugin",
diff --git a/node_modules/gustwind/esm/plugins/twind/mod.js b/node_modules/gustwind/esm/plugins/twind/mod.js
index 3176b6d..5aab9c7 100644
--- a/node_modules/gustwind/esm/plugins/twind/mod.js
+++ b/node_modules/gustwind/esm/plugins/twind/mod.js
@@ -1,4 +1,4 @@
-import * as path from "path";
+import * as path from "node:path";
 import { extract, install } from "@twind/core";
 const plugin = {
     meta: {
