diff --git a/node_modules/gustwind/esm/gustwind-utilities/plugins.js b/node_modules/gustwind/esm/gustwind-utilities/plugins.js
index e2cfaeb..bcbb3ab 100644
--- a/node_modules/gustwind/esm/gustwind-utilities/plugins.js
+++ b/node_modules/gustwind/esm/gustwind-utilities/plugins.js
@@ -1,5 +1,5 @@
-import { join as joinPath } from "path";
-import process from "process";
+import { join as joinPath } from "node:path";
+import process from "node:process";
 const DEBUG = process.env.DEBUG === "1";
 async function importPlugins({ cwd, initialImportedPlugins, pluginDefinitions, outputDirectory, initLoadApi, mode, }) {
     const loadedPluginDefinitions = [];
diff --git a/node_modules/gustwind/esm/plugins/breezewind-renderer/mod.js b/node_modules/gustwind/esm/plugins/breezewind-renderer/mod.js
index 4d83f7c..3d33941 100644
--- a/node_modules/gustwind/esm/plugins/breezewind-renderer/mod.js
+++ b/node_modules/gustwind/esm/plugins/breezewind-renderer/mod.js
@@ -1,5 +1,5 @@
 import * as dntShim from "../../_dnt.shims.js";
-import * as path from "path";
+import * as path from "node:path";
 import breezewind from "../../breezewind/mod.js";
 import { applyUtilities } from "../../breezewind/applyUtility.js";
 import * as breezeExtensions from "../../breezewind/extensions.js";
diff --git a/node_modules/gustwind/esm/plugins/config-router/mod.js b/node_modules/gustwind/esm/plugins/config-router/mod.js
index f1a7e10..60c63bc 100644
--- a/node_modules/gustwind/esm/plugins/config-router/mod.js
+++ b/node_modules/gustwind/esm/plugins/config-router/mod.js
@@ -1,4 +1,4 @@
-import * as path from "path";
+import * as path from "node:path";
 import { expandRoute, expandRoutes } from "./expandRoutes.js";
 import { flattenRoutes } from "./flattenRoutes.js";
 import { trim } from "../../utilities/string.js";
diff --git a/node_modules/gustwind/esm/plugins/copy/mod.js b/node_modules/gustwind/esm/plugins/copy/mod.js
index 5567b4b..c5fec1f 100644
--- a/node_modules/gustwind/esm/plugins/copy/mod.js
+++ b/node_modules/gustwind/esm/plugins/copy/mod.js
@@ -1,4 +1,4 @@
-import * as path from "path";
+import * as path from "node:path";
 const plugin = {
     meta: {
         name: "gustwind-copy-plugin",
diff --git a/node_modules/gustwind/esm/plugins/meta/mod.js b/node_modules/gustwind/esm/plugins/meta/mod.js
index 3462745..e1f7772 100644
--- a/node_modules/gustwind/esm/plugins/meta/mod.js
+++ b/node_modules/gustwind/esm/plugins/meta/mod.js
@@ -1,4 +1,4 @@
-import * as path from "path";
+import * as path from "node:path";
 const plugin = {
     meta: {
         name: "gustwind-meta-plugin",
diff --git a/node_modules/gustwind/esm/plugins/og/mod.js b/node_modules/gustwind/esm/plugins/og/mod.js
index 56ee1c9..b56dc0e 100644
--- a/node_modules/gustwind/esm/plugins/og/mod.js
+++ b/node_modules/gustwind/esm/plugins/og/mod.js
@@ -1,4 +1,4 @@
-import * as path from "path";
+import * as path from "node:path";
 // Note that this depends on --allow-ffi
 import sharp from "sharp";
 import { htmlToBreezewind } from "../../htmlisp/mod.js";
diff --git a/node_modules/gustwind/esm/plugins/pagefind/mod.js b/node_modules/gustwind/esm/plugins/pagefind/mod.js
index c1dd3bb..41c56df 100644
--- a/node_modules/gustwind/esm/plugins/pagefind/mod.js
+++ b/node_modules/gustwind/esm/plugins/pagefind/mod.js
@@ -1,4 +1,4 @@
-import * as path from "path";
+import * as path from "node:path";
 import * as pagefind from "pagefind";
 const plugin = {
     meta: {
diff --git a/node_modules/gustwind/esm/plugins/sitemap/mod.js b/node_modules/gustwind/esm/plugins/sitemap/mod.js
index 5f3e900..09d0c9b 100644
--- a/node_modules/gustwind/esm/plugins/sitemap/mod.js
+++ b/node_modules/gustwind/esm/plugins/sitemap/mod.js
@@ -1,4 +1,4 @@
-import * as path from "path";
+import * as path from "node:path";
 import { generateSitemap, sitemapToXML, } from "../../deps/deno.land/x/sitemap@v1.1.1/mod.js";
 // Note that this works only in production mode for now!
 const plugin = {
diff --git a/node_modules/gustwind/esm/plugins/twind/mod.js b/node_modules/gustwind/esm/plugins/twind/mod.js
index 7a575f1..9c44c65 100644
--- a/node_modules/gustwind/esm/plugins/twind/mod.js
+++ b/node_modules/gustwind/esm/plugins/twind/mod.js
@@ -1,4 +1,4 @@
-import * as path from "path";
+import * as path from "node:path";
 import { extract, install } from "@twind/core";
 const plugin = {
     meta: {
